src/Nox.Generator/Presentation/Api/OData/EntityController.TranslationsGenerator.cs

```diff
@@ -0,0 +1,54 @@
+﻿using System.Linq;
+using Microsoft.CodeAnalysis;
+
+using Nox.Generator.Common;
+using Nox.Solution;
+
+namespace Nox.Generator.Presentation.Api.OData;
+
+internal class EntityControllerTranslationsGenerator : EntityControllerGeneratorBase
+{
+    public override void Generate(
+        SourceProductionContext context,
+        NoxCodeGenConventions codeGeneratorState,
+        GeneratorConfig config, System.Action<string> log,
+        string? projectRootPath)
+    {
+        context.CancellationToken.ThrowIfCancellationRequested();
+
+        if (codeGeneratorState.Solution.Domain is null)
+        {
+            return;
+        }
+
+        const string templateName = @"Presentation.Api.OData.EntityController.Translations";
+
+        foreach (var entity in codeGeneratorState.Solution.Domain.Entities)
+        {
+            context.CancellationToken.ThrowIfCancellationRequested();
+
+            if (!entity.IsLocalized)
+            {
+                continue;
+            }
+            
+            var entityAttributesToLocalize = entity.Attributes
```

Comment:
see EntityExtensions and GetLocalizedAttributes

Reply:
Desired changes has been applied